participants/tali.md

```diff
@@ -0,0 +1,8 @@
+# tali ✨
+
+![](https://pbs.twimg.com/media/Dpx7aOyV4AAQbhd.jpg:large)
+
+- [tweets (don't follow me I'm really boring (but also please do))]()
```

Comment:
OH NO RIP I forgot the link somehow lmao this is what happens when u rush art @lorderikir